code/game/objects/items/hand_item.dm

```diff
@@ -68,6 +68,11 @@
 	blown_kiss.fire()
 	qdel(src)
 
+/obj/item/kisser/attack_self(mob/living/user)
+	to_chat(usr, "<span class='notice'>You lower your kiss-blowing hand!</span>")
+	qdel(src)
+	return
```

Comment:
Again, I personally think from an user perspective it makes more sense to retain the drop functionality to cancel the emote, rather than using it.